src/App.vue

```diff
@@ -163,4 +175,66 @@ li {
   }
 }
 
+// Footer styles.
+.site-footer {
+  display: flex;
+  flex-direction: column;
+  padding: 15px 30px 20px 30px;
+  background: $background3;
+  height: 294px;
+  width: 100%;
+  @media (min-width: $bp-medium) {
+    height: 167px;
```

Comment:
Same, no need for a fixed row. Let the inner elements define the container height.